ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/zones/techpreview/provision/cucushift-installer-rehearse-vsphere-ipi-zones-techpreview-provision-chain.yaml

```diff
@@ -1,9 +1,18 @@
 chain:
   as: cucushift-installer-rehearse-vsphere-ipi-zones-techpreview-provision
```

Comment:
I guess we can remove the provision/deprovision chain, just keep install workflow yaml and set `FEATURE_SET` there.